themes/openy_themes/openy_carnation/templates/openy_repeat/openy-repeat-schedule-dashboard--table.html.twig

```diff
@@ -154,42 +202,213 @@
       </div>
     </div>
   </div>
 
   {# modal > class info  #}
-  <div class="modal fade schedule-dashboard__modal schedule-dashboard__modal--class" tabindex="-1" role="dialog" aria-labelledby="schedule-dashboard__modal--class-link">
-    <div class="modal-dialog" role="document">
+  <div class="modal fade schedule-dashboard__modal schedule-dashboard__modal--class" tabindex="-1" role="dialog"
+       aria-labelledby="schedule-dashboard__modal--class-link">
+    <div class="modal-cover"></div>
+    <div class="modal-dialog px-2" role="document">
       <div class="modal-content">
 
         <div class="schedule-dashboard__modal--header">
-          <button type="button" class="close" data-dismiss="modal" aria-label="Close"><i class="fa fa-times" aria-hidden="true"></i></button>
+          <h5 class="modal-title">Activity details</h5>
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close"><i class="fa fa-times"
+                                                                                         aria-hidden="true"></i>
+          </button>
         </div>
 
         <div class="schedule-dashboard__modal--body">
-          <table class="table">
-            <thead>
-            <tr>
-              <th>
-                <h4><strong>${ classPopup.title }</strong></h4>
-              </th>
-            </tr>
-            </thead>
-            <tbody>
-            <tr>
-              <td>${ classPopup.description }</td>
-            </tr>
-            </tbody>
-          </table>
+          <div class="header-border p-3">
+            <h4><strong>${ classPopup.title }</strong></h4>
+            <p class="m-b">${ classPopup.description }</p>
+          </div>
+
+          <div v-for="(item, index) in classPopup.schedule" class="schedules-data__row">
+            <div class="col-md-2 col-sm-6 col-xs-6">
+              {# date #}
+              <div class="date-column" v-cloak>
+                <strong>${ item.short_date }</strong>
+                <p><span>${ item.day_name }</span></p>
+              </div>
+            </div>
+
+            <div class="col-md-2 col-sm-6 col-xs-6">
+              {# time #}
+              <div class="time-column" v-cloak>
+                <strong class="time">${ item.time_start }</strong>
+                <p>
+                  <span v-if="item.duration_hours > 0">${ item.duration_hours }{{ 'h'|t }}</span>
+                  <span v-if="item.duration_minutes > 0">${ item.duration_minutes }{{ 'min'|t }}</span>
+                </p>
+              </div>
+            </div>
+
+            <div class="col-md-3 col-sm-12 col-xs-12">
+              {# location #}
+              <div v-cloak class="location-column">
+                <strong>${ item.location }</strong>
+                <p><span v-if="item.room">${ item.room }</span></p>
+              </div>
+            </div>
+
+            <div class="col-md-3 col-sm-6 col-xs-6">
+              {{ dump(item.instructor) }}
```

Comment:
@shuklina Tnx for your work, we are looking for the fix too. Looks like you forgot to remove twig debug call here.

Reply:
Thank you for your review. I fixed this.